### ISSUE TITLE
Fast image 8.3.4 does not cache

### ISSUE THREAD

**Issue by emclab** [DylanVann/react-native-fast-image#769]
Here is a code loading an image from cloud with React Native 0.62.3/RN Fast Image 8.3.4:

     import { Col, Row, Grid } from 'react-native-easy-grid';
     import FastImage from 'react-native-fast-image';

     return (  
        <TouchableOpacity onPress={() => {navigation.navigate("Item Detail", item={item} )}}>
            <Grid style={styles.grid}>
                <Col size={1} >
                    <FastImage source={{uri:item.artimages[0].path}} style={styles.itemImage}/>  //<<==path is the cloud URL of the image. The display size of image is 40x40 and the size of image is less than 1MB.            
                </Col>
                <Col size={4}>
                    <Row size={4}>
                        <Text style={styles.titleText}>
                            {item.name}
                        </Text>
                    </Row>
                    <Row size={4}>
                        <Text style={styles.titleText}>
                            {item.description}
                        </Text>
                    </Row>
                </Col>
            </Grid>            
        </TouchableOpacity>
    );

The path is embedded with expiration and signature for authentication of the cloud image storage. Here is an URL example:

      https://myImage.aliyuncs.com/fOqMlr3Hc_7.jpg?Expires=1613253127&OSSAccessKeyId=TMP.3KhSceqrVN...VnjHh9DMq1dBZC&Signature=oponRvpd...EzylDU%D

What I notice on `Android emulator` is that it take about 5 second to load the image  each and every time. It seems that fast image is not caching. My understanding is that fast image caches by default. What causes Fast Image not caching the image here?

**Comment by ourfamilygithub** [comment]
same here

**Comment by emclab** [comment]
Any solution to the problem above? FastImage is not caching for any of the images residing on aliyuncs.com.

**Comment by dandre-hound** [comment]
yeah, same thing - android images just aren't caching. I'm running a debug build on a phone so it's not specific to the emulator. I'm not quite sure how to query the cache on android. @DylanVann any thoughts here? I'm happy to do some digging and create a repro example but I don't know if I should just log what's in the cache, run the android profiler, etc.

**Comment by emclab** [comment]
Hi, seems like this issue is not getting any attention. Are there other image solutions that have proper caching?
